global-test.jboss-eap.openshift.properties

```diff
@@ -7,18 +7,18 @@
 ###############################################
 
 # Databases
-intersmash.mysql.image=quay.io/centos7/mysql-80-centos7:centos7
-intersmash.postgresql.image=quay.io/centos7/postgresql-13-centos7:centos7
+intersmash.mysql.image=registry.redhat.io/rhel10/mysql-84:latest
```

Comment:
@fabiobrz same question as above: is that intended? I mean switching from quay.io to registry.redhat.io ...

Reply:
See my reply https://github.com/Intersmash/intersmash-tests/pull/41#discussion_r2409989855